api/src/API/Controllers/PublicRacesController.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using RaceResults.Common.Models;
+using RaceResults.Data.Core;
+
+namespace RaceResults.Api.Controllers
+{
+    [ApiController]
+    [Route("races/public")]
+    public class PublicRacesController : ControllerBase
```

Comment:
Maybe we should merge this with the existing races controller. Ultimately, race creation, updates and deletion are the same for all races. We could have an additional action for adding this unauthorized endpoint within the same controller

Reply:
I'm not really the expert on controller design, so I just hacked something together to get the routes working. Do you think you can do this as refactoring work later?